mysqld_exporter.go

```diff
@@ -56,10 +58,6 @@ var (
 		"collect.info_schema.innodb_metrics", false,
 		"Collect metrics from information_schema.innodb_metrics",
 	)
-	collectGlobalStatus = flag.Bool(
```

Comment:
It really boils down to whether there's someone using this code as a library that may want a different operational mode.

If it's just a random tuning variable, inside the file itself is fine. The node exporter ones fall under this, as plumbing them all into main wouldn't make sense, and noone is using it as a library.

Prometheus has gone too far in the other direction and plumbed through everything to main, which just makes it harder to add flags. This has also lead to us having more hardcoded constants than we should have.